.github/workflows/release.yml

```
@@ -0,0 +1,35 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+        with:
+          # This makes Actions fetch all Git history so that Changesets can generate changelogs with the correct commits
+          fetch-depth: 0
+
+      - name: Setup Node.js 16
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: npm
+
+      - name: Install Dependencies
+        run: npm i
```

[suggest] What about making deps installation consistent with other workflows?

```suggestion
      - name: Install latest npm 
        run: npm install --global npm@latest 
  
      - name: Install dependencies 
        run: npm ci 
```

https://github.com/stylelint/stylelint/blob/2c2a57cdca138aa6afe225b90b3fa6d2110245fc/.github/workflows/testing.yml#L41-L45

---

👍